src/Navbar.js

```diff
@@ -100,6 +126,7 @@ class Navbar extends React.Component {
         bsClass,
         expanded,
         onToggle: this.handleToggle,
+        onSelect: this.handleSelect,
```

Comment:
sorry, one last thing – i think it might be better to make this just

```js
createChainedFunction(onSelect, toggleOnSelect && this.handleToggle)
```

I think it's pretty unlikely that users will specify both `toggleOnSelect` and `onSelect`, and I think it'd be a bit cleaner this way.

Otherwise, this is looking really good. Thanks!